### ISSUE TITLE
Better Timer

### ISSUE THREAD

**Issue by hollesse** [remotemobprogramming/mob#308]
In our team we had a conversation that it would be cool if mob has a "real" timer. That can be stopped if I do mob next and which could be reset if a timer is running and I set a new timer.

**Comment by gregorriegler** [comment]
I'm not used to the timer. Sounds reasonable to me. Should we slice those features? Or would you like to collect it in one

**Comment by hollesse** [comment]
Let's slice it. I think what has to be done is kind of similar, but you could also do just one and then the other

**Comment by hollesse** [comment]
I thought about this again. I think we should forget about the part of the real timer. I think this effort does not bring any benefit, as it is not necessary to be 100% correct. If someone is typist for 10 minutes and 10 seconds instead of exactly 10 minutes this is okay IMHO. What do you think @gopp lol

**Comment by gregorriegler** [comment]
Then I didn't understand it entirely. Please provide an example for what you mean with the real timer.

**Comment by hollesse** [comment]
I just saw that I did not answer here. With a real timer I meant that it is not just a sleep which can take a bit longer than the seconds told to sleep. I thought about comparing with the systemclock how many time is gone. I am not sure if this would be an efficient way. But in my opinion it is not necessary as it does not make any difference if some is typist for 10 minutes an 10 seconds instead of 10 minutes an 0 seconds. This would be just wasted time to implement this.

One thing that has come to my mind now would be to implement the timer functionality with sleep and the sound and the notifications directly in go and start this go application in the background. There we could also implement a good way to abort a timer. This is something we could think about. But I think than we would need something like a OS specific daemon that runs in the background.
